site/spa/index.tsx

```diff
@@ -5,6 +5,7 @@
 
 import * as React from 'react';
 import * as ReactDOM from 'react-dom/client';
+import * as ReactGA from 'react-ga4';
```

Comment:
@ribose-jeffreylau Nice, I like that the lib has no dependencies, we could use it probably.

We don’t need to send a bunch of events, just initialize and record the visit to begin with…